GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Delta.cfg

```diff
@@ -746,25 +759,16 @@
     @maxTemp = 773.15
     %skinMaxTemp = 873.15
 
+    !MODULE[ModuleFuelTanks],*{}
+
     MODULE
     {
         name = ModuleFuelTanks
-        type = ServiceModule
-        volume = 58782.54
+        type = Cryogenic
```

Comment:
Same sort of question as above about appropriate tank choice. One way to tell is usage, how long did this stage ever coast and what kind of boiloff did it experience then? Also, a stage with a more insulated tank generally has a higher dry mass than the alternative.